### PR TITLE
Pbi 347 css animations

### PR DESCRIPTION
**Pull request by adphillips2017**
Issue #347

### CODE REVIEW

File: projects/ccf-rui/src/app/shared/components/visibility-toggle/visibility-toggle.component.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { Component, HostBinding, Input, Output, EventEmitter } from '@angular/co
 @Component({
   selector: 'ccf-visibility-toggle',
   templateUrl: './visibility-toggle.component.html',
-  styleUrls: ['./visibility-toggle.component.scss']
+  styleUrls: ['./visibility-toggle.component.scss'],
```
Comment (bherr2) [inline projects/ccf-rui/src/app/shared/components/visibility-toggle/visibility-toggle.component.ts:10]: Unnecessary change

**Review by bherr2 — CHANGES_REQUESTED** [review]
* [ ] Talked with lecross and he suggests speeding up the transitions from 1s to 0.6s. He also says the transition values are in the spec.
* [ ] Could you also add a ripple effect to the buttons like you see on the radio buttons and toggle switches? Let's give all buttons a ripple effect, unless they already have an on-press animation.

**Comment by bherr2** [comment]
The radius of the ripple causes scrollbars to appear randomly. Reduce the radius and have the ripple come from the middle always (I think that's a setting). Please spend some real time to make sure it looks and acts decent in both Firefox and Chrome.

**Comment by adphillips2017** [comment]
> The radius of the ripple causes scrollbars to appear randomly. Reduce the radius and have the ripple come from the middle always (I think that's a setting). Please spend some real time to make sure it looks and acts decent in both Firefox and Chrome.

Okay I made the ripple effect centered.  I'm not seeing any scrollbars appear when using any of the reset icons on Firefox or Chrome. If you are still seeing the scrollbar appear, there's a property I can change to set the ripple radius.

**Comment by bherr2** [comment]
> > The radius of the ripple causes scrollbars to appear randomly. Reduce the radius and have the ripple come from the middle always (I think that's a setting). Please spend some real time to make sure it looks and acts decent in both Firefox and Chrome.
> 
> Okay I made the ripple effect centered. I'm not seeing any scrollbars appear when using any of the reset icons on Firefox or Chrome. If you are still seeing the scrollbar appear, there's a property I can change to set the ripple radius.

Yep, that works. One minor thing and then this will be done: you forgot to add ripple to the reset button on the stage.

**Comment by adphillips2017** [comment]
> > > The radius of the ripple causes scrollbars to appear randomly. Reduce the radius and have the ripple come from the middle always (I think that's a setting). Please spend some real time to make sure it looks and acts decent in both Firefox and Chrome.
> > 
> > 
> > Okay I made the ripple effect centered. I'm not seeing any scrollbars appear when using any of the reset icons on Firefox or Chrome. If you are still seeing the scrollbar appear, there's a property I can change to set the ripple radius.
> 
> Yep, that works. One minor thing and then this will be done: you forgot to add ripple to the reset button on the stage.

It's there, it's just almost impossible to see because of the black background.  I can change the color to something else easy enough, though.  Should I ask Leonard?

**Comment by bherr2** [comment]
> > > > The radius of the ripple causes scrollbars to appear randomly. Reduce the radius and have the ripple come from the middle always (I think that's a setting). Please spend some real time to make sure it looks and acts decent in both Firefox and Chrome.
> > > 
> > > 
> > > Okay I made the ripple effect centered. I'm not seeing any scrollbars appear when using any of the reset icons on Firefox or Chrome. If you are still seeing the scrollbar appear, there's a property I can change to set the ripple radius.
> > 
> > 
> > Yep, that works. One minor thing and then this will be done: you forgot to add ripple to the reset button on the stage.
> 
> It's there, it's just almost impossible to see because of the black background. I can change the color to something else easy enough, though. Should I ask Leonard?

Ok, just change the color using your best judgement. Leonard will review as a whole at the end of the sprint.

**Review by bherr2 — APPROVED** [review]

**Pull request merged**